missing "dist" directory

I am a bit confused on the proper way to import funcunit and syn into my project.
I am using es6 modules with webpack which is attempting to load funcunit via cjs build:
I installed funcunit with:
`npm install funcunit@3.1.0-pre.1`
when I perform my build I get an error:
`ERROR in ./~/funcunit/dist/cjs/funcunit.js
Module not found: Error: Cannot resolve module 'syn' in .../node_modules/funcunit/dist/cjs`
so i installed "syn"
`npm install syn`
but I still received the same error, I then saw that the git tag v0.2.0 had a "dist" folder so I installed directly from github using the v0.2.0 tag:
`npm install git://github.com/bitovi/syn.git#v0.2.0`
this worked, and now funcunit is working.
Should syn master branch have a dist folder?

Thanks for the great package, loving funcunit.

--- a/.npmignore
+++ b/.npmignore
@@ -8,3 +8,4 @@ Gruntfile.js
 .jshintrc
 .travis.yml
 .editorconfig
+!dist/

